rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -2266,6 +2266,39 @@ void getProposedFederationAddress(MessageCall.MsgType msgType, ActivationConfig
         }
     }
 
+    @ParameterizedTest()
+    @MethodSource("msgTypesAndActivations")
+    void getProposedFederatorPublicKeyOfType(MessageCall.MsgType msgType, ActivationConfig activationConfig) throws VMException {
+        Transaction rskTxMock = mock(Transaction.class);
+        doReturn(true).when(rskTxMock).isLocalCallTransaction();
+
+        int federatorIndex = 1;
+        KeyType keyType = KeyType.BTC;
```

Comment:
would it be worth testing it for other key types?

Reply:
I think this should be good, we are just veryfing that the bridge method gets called and executed no matter the params.

Reply:
makes sense :)